terraform/aws/development-infrastructure/nbs-legacy/main.tf

```diff
@@ -116,9 +116,12 @@ Set-Location -Path "D:\wildfly-10.0.0.Final\bin\service"
 #Set-Service Wildfly -StartupType Automatic
 sc.exe config Wildfly start= delayed-auto
 Start-Service Wildfly
+Restart-Computer -Force
 </powershell>
 EOF
 
+    depends_on = [module.db]
```

Comment:
This is ok as long as we are deploying these things in a single module. If we ever split out the DB, this will change too.